packetbeat/docs/configuration.asciidoc

```diff
@@ -199,6 +200,34 @@ you use this setting, it's your responsibility to keep the BPF filters in sync w
 ports defined in the `protocols` section.
 
 
+[[configuration-flows]]
+=== Flows
+
+The `flows` section contains configuration options for bidirectional network
+flows. If section is missing from configuration file, network flows are
+disabled.
```

Comment:
What about the BPF filter? Flows will work only for what passes the current BPF, right?

I wonder if it wouldn't be worth being able to configure TCP/UDP ports on which flows are explicitely enabled.